plugins/modules/current_container_facts.py

```diff
@@ -101,6 +106,23 @@ def main():
             container_id = cgroup_name
             container_type = 'github_actions'
 
+    if not container_id and os.path.exists(mountinfo_path):
+        with open(mountinfo_path, 'rb') as f:
+            contents = f.read().decode('utf-8')
+
+        for line in contents.splitlines():
+            parts = line.split()
+            if len(parts) >= 5 and parts[4] == '/etc/hostname':
+                m = re.match('/var/lib/docker/containers/([a-f0-9]+)/hostname', parts[3])
```

Comment:
Testing on docker version 20.10.21, it starts with `/@/var/...`
I'm using Arch Linux (kernel 6.0.10-arch2-1) with cgroups v2 if that matters

Reply:
https://gist.github.com/kristianheljas/b20df29c43ab96fddbd7acd36d45a965

Reply:
Oh, the `/@` is btrfs subvolume - silly me! This can probably be longer as well.
Matching only the end of the should still be strict enough to account for that.

Reply:
Implemented in 34e94e078c7d1624475d9f95700bdfdd545250ef.

Reply:
/resolved